### ISSUE TITLE
unsupported UI options (copied from documentation)

### ISSUE THREAD

**Issue by pixelass** [patternplate/patternplate#224]
Theme copied from [https://patternplate.github.io](https://patternplate.github.io/pattern/themes?guides-enabled=true&advanced-enabled=true&components-enabled=true) causes error due to unsupported options.

```
13:10 $ node_modules/.bin/patternplate --version
2.5.2
```

```js
// patternplate.config.js
module.exports = {
  ui: {
    logo: `
      <svg width="30" height="30" viewBox="0 0 24 24">
        <rect width="30" height="30" fill="currentColor"/>
      </svg>
    `,

    // Fonts
    fontDefault: '-apple-system, BlinkMacSystemFont, "Segoe UI", Helvetica, Arial, sans-serif',
    fontHeadline: '-apple-system, BlinkMacSystemFont, "Segoe UI", Helvetica, Arial, sans-serif',
    fontCode: 'SFMono-Regular, Consolas, Liberation Mono, Menlo, Courier, monospace',

    // Global colors
    colorActive: "rgb(51, 153, 255)",
    colorError: "rgb(250, 63, 69)",
    colorWarning: "rgb(255, 189, 46)",
    colorInfo: "rgb(80, 179, 221)",
    colorSuccess: "rgb(74, 165, 74)",

    // Dark context colors
    colorBackgroundDark: "rgb(8, 15, 23)",
    colorBackgroundSecondaryDark: "rgb(11, 23, 34)",
    colorBackgroundSecondaryTertiaryDark: "rgb(11, 23, 34)",
    colorBorderDark: "rgb(11, 23, 34)",
    colorTextDark: "rgb(242, 242, 242)",
    colorRecessDark: "rgb(153, 153, 153)",

    // Light context colors
    colorBackgroundLight: "rgb(255, 255, 255)",
    colorBackgroundSecondaryLight: "rgb(246, 248, 250)",
    colorBackgroundTertiaryLight: "rgb(246, 248, 250)",
    colorBorderLight: "rgb(228, 228, 228)",
    colorTextLight: "rgb(68, 68, 68)",
    colorRecessLight: "rgb(106, 115, 125)"
  }
};
```

```
13:10 $ yarn start
yarn run v1.5.1
$ patternplate start

Invalid config at patternplate.config.js:
  options.ui['colorBackgroundSecondaryTertiaryDark']  is an invalid additional property
error An unexpected error occurred: "Command failed.
Exit code: 1
Command: sh
Arguments: -c patternplate start
Directory: /Users/greada/Workspace/Sinnerschrader/frontend-dev/frontend-dev-guide
Output:
".
info If you think this is a bug, please open a bug report with the information provided in "/Users/greada/Workspace/Sinnerschrader/frontend-dev/frontend-dev-guide/yarn-error.log".
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

**Comment by marionebl** [comment]
`colorBackgroundSecondaryTertiaryDark` should be `colorBackgroundTertiaryDark` here

https://github.com/patternplate/patternplate/blob/master/packages/components/src/themes/demo.js#L43

**Comment by niebuhrhood** [comment]
Hej folks,
i fixed it in #229
